packages/frontend/src/app/components/transactions/WalletConnectQR.tsx

```diff
@@ -69,7 +69,10 @@ function WalletConnectQR({
         const sessionNamespace = await approval();
         if (sessionNamespace) {
           setSession(sessionNamespace);
-          setAccount(sessionNamespace.namespaces.eip155.accounts[0].slice(9));
+          const accountAddress = sessionNamespace.namespaces.eip155.accounts[0]
+            .slice(9)
```

Comment:
```suggestion
            .slice(9) // split of 'ethereum:'  prefix
```

seems a bit to magical without a comment